docker/entrypoint_vlad.sh

```diff
@@ -140,4 +140,8 @@ if [ -f "${config_dir}/startup.sh" ]; then
     popd > /dev/null
 fi
 
+# Upgrade pip so everything installs correctly
+pip install --upgrade pip
```

Comment:
Hm. I thought I was updating pip and wheel in the base image build, but evidently not. Good catch, have just updated `Dockerfile.base` to cover this 😄